testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -10901,6 +10901,26 @@ void checkEndsWith(SqlOperatorFixture f0, FunctionAlias functionAlias) {
             + "requires extra delimiter argument", false);
   }
 
+  /** Tests the {@code SPLIT_PART} operator. */
```

Comment:
Add unit test to cover `REDSHIFT`? I notice this function is except for `REDSHIFT`.

Reply:
Added the test, addressed in latest PR, thanks!